### ISSUE TITLE
Batch Normalization (BN) | LogM's Blog

### ISSUE THREAD

**Issue by imLogM** [imLogM/blog_gitment#6]
https://imlogm.github.io/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/batch-normalization/
